apollo-kotlin-sample/src/main/java/com/apollographql/apollo/kotlinsample/KotlinSampleApp.kt

```diff
@@ -0,0 +1,61 @@
+package com.apollographql.apollo.kotlinsample
+
+import android.app.Application
+import com.apollographql.apollo.ApolloClient
+import com.apollographql.apollo.api.Operation
+import com.apollographql.apollo.api.ResponseField
+import com.apollographql.apollo.cache.normalized.CacheKey
+import com.apollographql.apollo.cache.normalized.CacheKeyResolver
+import com.apollographql.apollo.cache.normalized.sql.ApolloSqlHelper
+import com.apollographql.apollo.cache.normalized.sql.SqlNormalizedCacheFactory
+import okhttp3.OkHttpClient
+
+@Suppress("unused")
+class KotlinSampleApp : Application() {
+
+  companion object {
+    // https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/
+    const val GITHUB_OAUTH_TOKEN = "your_token"
+  }
+
+  private val baseUrl = "https://api.github.com/graphql"
+  lateinit var apolloClient: ApolloClient
+    private set
```

Comment:
I would make this a `lazy` property so you don't delay application start. Plus, it would encapsulate the client creation nicely.

Reply:
Fixed